ass1-sass/sass/pages/_home.scss

```diff
@@ -0,0 +1,444 @@
+//scss-lint:disable NestingDepth SelectorDepth
```

Comment:
@priyanka0901 Why you have disabled NestingDepth and SelectorDepth if you are not using scss-lint in grunt file. We need to avoid big nesting when we are not using any library.